execution-engine/src/lib.rs

```diff
@@ -61,7 +60,7 @@ pub fn execute(
         }
         ExecutionStrategy::JIT => {
             cfg_if::cfg_if! {
-                if #[cfg(feature = "std")] {
+                if #[cfg(any(feature = "std", feature = "nitro"))] {
```

Comment:
Would it make more sense to instead make the `nitro` feature define the `std` feature instead?

Semantically speaking, the Nitro environment should be a `std` environment.

Reply:
Yes, I think so, though I think this should be part of a much-wider look at our project `Cargo.toml` files and the features that they're using.